jax/core.py

```diff
@@ -1287,10 +1289,15 @@ def pp_eqn_compact(primitive_name: str, params: Dict) -> PrettyPrint:
 
 def pp_eqn(eqn: JaxprEqn) -> PrettyPrint:
   lhs = pp_vars(eqn.outvars)
-  pp_subexpr = pp('')
-  return (pp('{} = '.format(lhs)) >>
-          pp(eqn.primitive.name) >> pp_kv_pairs(sorted(eqn.params.items()))
-          >> pp(' ') >> pp(pp_vars(eqn.invars))) + pp_subexpr
+  pp_lhs = pp(f'{lhs} =')
+  pp_rhs = (pp(eqn.primitive.name) >>
+            pp_kv_pairs(sorted(eqn.params.items())) >> pp(' ') >>
+            pp(pp_vars(eqn.invars)))
+  if len(lhs) <= pp_jaxpr_eqn_lhs_wrap_threshold_length:
+    pp_eqn = pp_lhs >> pp(' ') >> pp_rhs
+  else:
+    pp_eqn = pp_lhs + pp_rhs.indent(2)
+  return pp_eqn + pp('')
```

Comment:
```suggestion
  return pp_eqn
```

That is vestigial, I believe!